### PR TITLE
Fix vendor table build in BUILD.gn for nonsemantic.vulkan.debuginfo.100

### LINKED ISSUE #4374: Missing inc file in GN build after PR 4362
We're getting the following downstream while building ANGLE dependencies:
`ninja: error: '../../third_party/vulkan-deps/spirv-headers/src/include/spirv/unified1/extinst.nonsemantic.vulkan.debuginfo.100.grammar.json', needed by 'gen/third_party/vulkan-deps/spirv-tools/src/nonsemantic.vulkan.debuginfo.100.insts.inc', missing and no known rule to make it`

https://chromium-review.googlesource.com/c/angle/angle/+/3020508

It looks like that inc file is generated and support needs to be added for GN.

### PR DESCRIPTION
**Pull request by greg-lunarg**
Fixes #4374

### CODE REVIEW

**Comment by cnorthrop** [comment]
With local testing, I'm getting the following with this change:
```
FAILED: gen/third_party/vulkan-deps/spirv-tools/src/nonsemantic.vulkan.debuginfo.100.insts.inc 
python3 ../../third_party/vulkan-deps/spirv-tools/src/utils/generate_grammar_tables.py --extinst-vendor-grammar source/extinst.nonsemantic.vulkan.debuginfo.100.grammar.json --vendor-insts-output gen/third_party/vulkan-deps/spirv-tools/src/nonsemantic.vulkan.debuginfo.100.insts.inc --vendor-operand-kind-prefix VKDEBUG100_
Traceback (most recent call last):
  File "/path/to/angle/out/AndroidDebug/../../third_party/vulkan-deps/spirv-tools/src/utils/generate_grammar_tables.py", line 874, in <module>
    main()
  File "/path/to/angle/out/AndroidDebug/../../third_party/vulkan-deps/spirv-tools/src/utils/generate_grammar_tables.py", line 862, in main
    with open(args.extinst_vendor_grammar) as json_file:
FileNotFoundError: [Errno 2] No such file or directory: 'source/extinst.nonsemantic.vulkan.debuginfo.100.grammar.json'
```

File: BUILD.gn [hunk 1]
```diff
@@ -224,6 +224,30 @@ template("spvtools_vendor_table") {
...
+    script = "utils/generate_grammar_tables.py"
+
+    name = invoker.name
+    extinst_vendor_grammar =
+        "source/extinst.${name}.grammar.json"
+    extinst_file = "${target_gen_dir}/${name}.insts.inc"
+
+    args = [
+      "--extinst-vendor-grammar",
+      extinst_vendor_grammar,
```
Comment (cnorthrop) [inline BUILD.gn:240]: Adding a rebase to this line compiles on my end:
```
rebase_path(extinst_vendor_grammar, root_build_dir),
```

**Review by cnorthrop — CHANGES_REQUESTED** [review]

File: BUILD.gn (hunk 1 above)
Comment (greg-lunarg) [reply BUILD.gn:240]: Excellent! Thanks. That's what I was looking for. I will add it to my PR and retry.

**Review by cnorthrop — APPROVED** [review]

**Review by jaebaek — APPROVED** [review]
Great! Thank you for fixing the issue!

**Pull request merged**